src/pages/komodo-defi-framework/api/v20-dev/non_fungible_tokens/update_nft/index.mdx

```diff
@@ -42,11 +44,13 @@ These coins can be activated using the [enable\_eth\_with\_tokens](/komodo-defi-
     "method": "update_nft",
     "mmrpc": "2.0",
     "params": {
-      "chains": [
+      "chains": [ 
+        "BSC",
         "POLYGON"
       ],
       "url": "https://moralis-proxy.komodo.earth",
       "url_antispam": "https://nft.antispam.dragonhound.info",
+    }
       "proxy_auth": false
```

Comment:
we started to use "komodo_proxy" field instead of "proxy_auth" (and also instead of `gui_auth` for eth/erc20 activation rpcs)

it was updated in this PR https://github.com/KomodoPlatform/komodo-defi-framework/pull/2173

please change proxy field for  `update_nft` and `refresh_nft_metadata` RPCs.

As for `gui_auth`, I suppose you also can update it.

`"komodo_proxy"` is optional in up to date dev branch